verilog/CST/expression.cc

```diff
@@ -98,6 +98,19 @@ const verible::Symbol* GetConditionExpressionFalseCase(
   return GetSubtreeAsSymbol(condition_expr, NodeEnum::kConditionExpression, 4);
 }
 
+const verible::TokenInfo* GetUnaryPrefixOperator(
+    const verible::Symbol& symbol) {
+  const auto& children = verible::SymbolCastToNode(symbol).children();
+  const auto leaf_symbol = children.front().get();
+  return &verible::down_cast<const verible::SyntaxTreeLeaf*>(leaf_symbol)
```

Comment:
Do we know that `leaf_symbol` we get is a `SyntaxTreeLeaf` so that we can safely cast ?

Reply:
I assumed every kUnaryPrefixExpression has the following form

```
Node @0 (tag: kUnaryPrefixExpression) {
                        Leaf @0 /* ... */
                        Node @1 /* ... /*
```

So I guess that if we're sure the current node is a kUnaryPrefixExpression we can assume the cast will work always? Let me know if I'm wrong.

Reply:
At the time of the writing of this comment, we only got `verible::Symbol`, so didn't know that yet.